src/assets/stylesheets/shared/_overrides.less

```diff
@@ -21,9 +21,6 @@
 .cards-pf { background-color: @color-pf-black-200; }
 .list-pf-item { background-color: @color-pf-white; }
 .blank-slate-pf {
-  /* stylelint-disable */
-  /* required to overwrite PatternFly blank slate to achieve new default white background */
-  background-color: @color-pf-white !important;
+  background-color: @color-pf-white;
```

Comment:
@SMahil this needs to stay as originally written. Removing the `!important` has negative affects on other parts of the application.